modules/management-console-agent/src/main/java/org/apache/ignite/agent/ws/RetryableSender.java

```diff
@@ -119,28 +119,28 @@ boolean sendInternal(String dest, List<T> elements) throws InterruptedException
     /**
      * @param element Element to send.
      */
-    public void send(String dest, T element) {
+    public void send(String dest, Object element) {
         if (element != null)
             addToQueue(new IgniteBiTuple<>(dest, Collections.singletonList(element)));
```

Comment:
Call `sendList`

Reply:
Why we should try to split on batches message with single element?